### ISSUE TITLE
confusion

### ISSUE THREAD

**Issue by Vvfshadow** [leeyeehoo/SiamRPN-VGG#4]
When I saw your paper,I thought you just replaced the backbone.....RPN was not mentioned in your paper...

**Comment by leeyeehoo** [comment]
yep. it's just a combination of our backbone and the rpn module.
